.ci/scripts/install-tools.bat

```diff
@@ -17,6 +17,7 @@ SET USERPROFILE=%OLD_USERPROFILE%
 
 echo "Upgrade chocolatey to latest version"
 choco upgrade chocolatey -y
+refreshenv
```

Comment:
do we still need this? I recall it was added in an effort to fix the docker-compose issues but it didn't help and it'll be addressed via https://github.com/elastic/beats/pull/38678 instead?

Reply:
You're right, thanks for catching that! I removed it in 9794cd3858